thatkitebot/cogs/funstuffcog.py

```diff
@@ -285,7 +286,16 @@ async def _uwuify(self, ctx: commands.Context, *, msg: str = None):
             # declare a new uwu object using the message id as seed
             uwu = uwuipy(seed)
             # convert the input string to ascii
-            msg = uwu.uwuify(unidecode(msg))
+            msg = unidecode(msg)
+
+            # if the user cant send images, make links not embed by surrounding them with <>
+            if not can_send_image(ctx):
+                links = r"(https?:\/\/[A-Za-z0-9\-._~!$&'()*+,;=:@\/?]+)"
```

Comment:
couldn't you just do `https?:\/\/\S+`

Reply:
Idk i copied this regex from stack overflow because I didnt want to fuck it  up